src/compiler/_namespaces/ts.moduleSpecifiers.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable simple-import-sort/exports */

```

Comment:
Maybe add an override in `.eslintrc.json` for all files part of `/_namespaces/` path, to disable this specific rule.

Reply:
That's not a bad idea, but I don't see us ever adding any more of these files anymore (in fact, I have plans to delete most of them)